e2e/api/steps.js

```diff
@@ -151,7 +151,7 @@ module.exports = {
     await validateEventPages(data[eventName]);
 
     await assertError('Upload', data[eventName].invalid.Upload.duplicateError,
-      'More than one Particulars of claim details added');
+      'You need to either upload 1 Particulars of claim only or enter the Particulars of claim text in the field provided. You cannot do both.');
```

Comment:
Please add some content to the ticket to factor in these changes about error. It currently does not mention anything about errors, only updating document field to collection